packages/rocketchat-ui-account/client/accountProfile.html

```diff
@@ -1,76 +1,155 @@
+									<div class="rc-input__wrapper">
+										<div class="rc-input__icon">
+											<svg class="rc-input__icon-svg rc-input__icon-svg--at">
+												<use xlink:href="#icon-at"></use>
+											</svg>
+										</div>
+										<input type="text" class="rc-input__element" name="username" id="username" value="{{username}}" {{ifThenElse canChange '' 'disabled'}}>
+									</div>
+								</label>
+								{{# unless canChange}}
+									<div class="rc-input__description">{{_ 'Username_Change_Disabled'}}</div>
+								{{/unless}}
+								{{#unless usernameAvaliable}}
+									<div class="rc-input__error">
+										<div class="rc-input__error-icon">
+											<svg class="rc-input__error-icon-svg rc-input__error-icon--warning">
+												<use xlink:href="#icon-warning"></use>
+											</svg>
+										</div>
+										<div class="rc-input__error-message">{{_ "Username_already_exist"}}</div>
+									</div>
+								{{/unless}}
 							</div>
-						{{/unless}}
+							{{/with}}
 						</div>
-					</div>
-					<div class="input-line">
-						<label for="password">{{_ "New_password"}}</label>
-						<div>
-						{{#if allowPasswordChange}}
-							<input type="password" name="password" id="password" />
-						{{else}}
-							<input type="password" name="password" id="password" disabled readonly="readonly" title="{{passwordChangeDisabled}}" placeholder="{{passwordChangeDisabled}}"/>
-						{{/if}}
+						<div class="rc-form-group rc-grid">
+							{{#with canChange=allowEmailChange}}
+							<div class="rc-input rc-w50 padded {{#if emailInvalid}}rc-input--error{{/if}}">
+									<label class="rc-input__label">
+										<div class="rc-input__title">{{_ "Email"}}</div>
+										<div class="rc-input__wrapper">
+											<input name="email" type="text" class="rc-input__element" placeholder="Type channel name" value="{{email}}" autocomplete="false" {{ifThenElse canChange '' 'disabled'}}>
+										</div>
+									</label>
+									{{# unless canChange}}
+										<div class="rc-input__description">{{_ 'Email_Change_Disabled'}}</div>
+									{{/unless}}
+							</div>
+							{{/with}}
+							<div class="rc-input rc-w50 padded">
+								{{#with canChange=allowPasswordChange}}
+									<label class="rc-input__label">
+										<div class="rc-input__title">{{_ "New_password"}}</div>
+										<div class="rc-input__wrapper">
+											<input name="password" type="password" class="rc-input__element" placeholder="Type channel name"  autocomplete="new-password" {{ifThenElse canChange '' 'disabled'}}>
+										</div>
+									</label>
+									{{# unless canChange}}
+										<div class="rc-input__description">{{_ 'Password_Change_Disabled'}}</div>
+									{{/unless}}
+								{{/with}}
+							</div>
 						</div>
 					</div>
 				</fieldset>
 				<fieldset>
-					{{> customFieldsForm hideFromForm=false formData=customFields}}
+					<!-- <legend style="margin-bottom: 2.5rem;">Custom Fields</legend> -->
```

Comment:
we have no config to customize the legend, and `custom fields`its a little confuse\useless